pennylane_lightning/src/gates/cpu_kernels/avx_common/README.md

```diff
@@ -0,0 +1,14 @@
+# Implementation of PennyLane-Lightning AVX2/512 kernels
```

Comment:
It would be great if you could provide complete and explicit examples, for each method, and how they come together.

Reply:
You can choose some small cases and show the process step-by-step.

Reply:
I used documentation to explain more details: https://docs.pennylane.ai/projects/lightning/en/stable/avx_kernels/implementation.html. Do you think we still need to add more explanation here?